spec/system/simple_query_spec.rb

```diff
@@ -44,6 +44,18 @@
       end
     end
 
+    context "when the query is a tel number but the API_KEY is invalid" do
+      let(:what) { "071 2222 777" }
+
+      it "raises an exception" do
+        expect { subject }.to raise_error(TelSearch::Error)
+      end
+
+      it "reports the error body in the TelSearch::Error body" do
+        expect { subject }.to raise_error (/API-Key is invalid/)
+      end
+    end
```

Comment:
I'd just use one VCR cassette, since we test the same thing (I noticed that we could do this in other specs as well, but this is outside of the scope of this PR).

Also we should set `API_KEY` explicitly to an invalid value, so re-recording of the VCR works:
```
let(:client) { TelSearch::Client.new('invalid-key') }
```